### DISCUSSION TITLE
Journal Week View

### DISCUSSION THREAD

**Discussion by OiAnthony** [toeverything/AFFiNE#6175] [Ideas]
### Description

Week picker may refer to capacities
Support accessing documents of the current week on the same page to facilitate users in summarizing their weekly work.
<img width="1471" alt="image" src="https://github.com/toeverything/AFFiNE/assets/62509433/98a02254-e9ec-4178-b27b-8394398be437">

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by OiAnthony** [comment]
拍了拍大佬 @CatsJuice

**Comment by almereyda** [comment]
This is also how LogSeq and Obsidian handle this.
